src/server/src/main/java/io/cassandrareaper/jmx/JmxConnectionFactory.java

```diff
@@ -163,4 +175,55 @@ private void decrementSuccessfullConnections(String host) {
       LOG.warn("Could not decrement JMX successfull connections counter for host {}", host, e);
     }
   }
+
+
```

Comment:
Could the following three methods (and the initSeedsJmxExecutor field) be encapsulated into a separate class.

If so then such a class could be AutoClosable so we could close the ExecutorService as a resource like
```
 private void initializeJmxSeedsForAllClusters() {
    LOG.info("Initializing JMX seed list for all clusters...");
    try (
         JmxConnectionsIntializer jmxConnectionsIntializer = new JmxConnectionsIntializer(context.jmxConnectionFactory);
         Timer.Context cx 
             = context.metricRegistry.timer(MetricRegistry.name(JmxConnectionFactory.class, "jmxConnectionsIntializer")).time()) {

      context.storage.getClusters()
          .parallelStream()
          .forEach(cluster -> jmxConnectionsIntializer.on(cluster));
    }
  }
```


(I think adding metrics is needed here, and i think inside `jmxConnectionsIntializer.on(cluster)` and inside `JmxProxy.connect(..)`)

Reply:
The UI and the REST API will be unavailable until the init phase is over, so we'll need there's no feedback we could give them anyway.